HW7/controller.py

```diff
@@ -0,0 +1,21 @@
+import view
+import model
+
+
+def run():
+    num = 1
+    while num > 0:
+        view.input_data()
+        num = int(input('Выберети действие: '))
+        if num == 1:
+            model.show_dict()
+        elif num == 2:
+            model.enter_cont()
+        elif num == 3:
+            model.find_cont()
+        elif num == 4:
+            model.del_cont()
+        elif num == 9:
+            model.del_file()
+        elif num == 0:
+            quit()
```

Comment:
Я говорил, что лучше не использовать такие функции для завершения работы программы. Если хотите завершить программу таким образом, то делайте так:
```suggestion
            sys.exit()
```

Reply:
И вообще чем вам простой `break` не угодил?

Reply:
Да и вообще, зачем вам `break` или что-либо другое, если у вас все равно завершится работа программы, если пользователь введёт ноль, потому что у вас в условии цикла `while` это прописано.